examples/hosted-wallet-kt/pfi/src/main/kotlin/data/Offerings.kt

```diff
@@ -0,0 +1,100 @@
+package data
+
+import tbdex.sdk.resources.Offering
+import tbdex.sdk.resources.OfferingData
+import tbdex.sdk.resources.PayinDetails
+import tbdex.sdk.resources.PayinMethod
+import tbdex.sdk.resources.PayoutDetails
+import tbdex.sdk.resources.PayoutMethod
+import tbdex.sdk.web5.BearerDid
+import tbdex.sdk.web5.PresentationDefinition
+import tbdex.sdk.web5.InputDescriptor
+import tbdex.sdk.web5.Constraints
+import tbdex.sdk.web5.Field
+import tbdex.sdk.web5.Filter
+
+class Offerings(private val bearerDid: BearerDid) {
+    private var offerings: MutableList<Offering> = mutableListOf()
+
+    init {
+        // this app currently always seeds the fake data
```

Comment:
i think eventually we should put these hardcoded things in a resource folder as a .json, will be easier to see the full picture / edit / swap out / test

Reply:
yeah I like this idea a lot, if you want to take the reigns then please do